tests/SigningTest.php

```diff
@@ -74,14 +73,20 @@ public function testSigv4HeaderSigning() {
             'credentials_provider' => $credentials_provider,
             'region' => self::SIGV4TEST_REGION,
             'service' => self::SIGV4TEST_SERVICE,
+            'date' => $date,
         ]);
         $http_request = new Request('GET', '/', [], ['Host' => 'example.amazonaws.com']);
         $this->assertNotNull($http_request, "Unable to create HttpRequest for signing");
         $signable = Signable::fromHttpRequest($http_request);
         $this->assertNotNull($signable, "Unable to create signable from HttpRequest");
 
-        Signing::signRequestAws($signable, $signing_config, function() {
-
+        Signing::signRequestAws($signable, $signing_config, function($signing_result, $error_code) use (&$http_request) {
```

Comment:
this looks like more than 80 characters and if it is should be split into more lines